v7/import_blogger/import_blogger.py

```diff
@@ -182,21 +187,19 @@ def import_item(self, item, out_folder=None):
         else:
             is_draft = False
 
-        self.url_map[link] = self.context['SITE_URL'] + '/' + \
-            out_folder + '/' + slug + '.html'
+        self.url_map[link] = os.path.join(
```

Comment:
You can’t use `os.path.join` for URLs (that would give backslashes on Windows), just use slashes manually or do os.path.join().replace(os.sep, '/')